docker-compose.yml

```diff
@@ -4,24 +4,20 @@ services:
     db:
         image: postgres:11.2-alpine
         ports:
-            - "5432"
+            - "54321:5432"
         environment:
             - POSTGRES_DB=spinta
             - POSTGRES_USER=admin
             - POSTGRES_PASSWORD=admin123
         volumes:
             - "./postgres-data:/var/lib/postgresql/data"
-    app:
-        build: .
+    mongo:
+        image: mongo:4.0.9-xenial
         ports:
-            - "8000:8000"
+            - "27017:27017"
         environment:
-            - SPINTA_BACKENDS_DEFAULT_DSN=postgresql://admin:admin123@db:5432/spinta
-            - SPINTA_MANIFESTS_DEFAULT_PATH=tests/manifest
-            - SPINTA_TEST_DATABASE_DSN=postgresql://admin:admin123@db:5432/spinta_tests
-        depends_on:
-            - db
+            - MONGO_INITDB_DATABASE=spinta
+            - MONGO_INITDB_ROOT_USERNAME=admin
+            - MONGO_INITDB_ROOT_PASSWORD=admin123
         volumes:
-            - ".:/opt/spinta"
-        stdin_open: true
-        tty: true
+            - "./mongo-data:/data/db"
```

Comment:
Why do you need data directory on host machine? Maybe it is better to let Docker handle it?

Reply:
For persistence between docker containers.

Reply:
`- "/data/db"` will do the same, except you don't need to specify a path on host machine.